Documentation/Installation/Index.rst

```diff
@@ -0,0 +1,44 @@
+.. include:: ../Includes.txt
+
+
+
+.. _installation:
+
+============
+Installation
+============
+
+Target group: **Administrators**
+
+This extension is part of the TYPO3 core.
```

Comment:
That's not the case in this manual. Please mention that this extension is a minimal version of the feature that will be released in core v10.

Reply:
just removed it completely